subscribers:
  
  queue_size: 5

  image_rgb:
    topic: /camera/color/image_raw

  image_depth:
    topic: /camera/depth/image_rect_raw

  camera_info:
    topic: /camera/color/camera_info

  points:
    topic: /camera/depth/color/points

publishers:
  
  queue_size: 1

  image_rgb:
    topic: /butia_vision/bvb/image_rgb_raw

  image_depth:
    topic: /butia_vision/bvb/image_depth_raw

  camera_info:
    topic: /butia_vision/bvb/camera_info

  points:
    topic: /butia_vision/bvb/points

servers:

  image_request:
    service: /butia_vision/bvb/image_request
